src/test/java/com/glovoapp/planningpoker/e2e/Clear_ShowVotes_ShareSessionLink/ShowVotesButtonTest.java

```diff
@@ -0,0 +1,40 @@
+package com.glovoapp.planningpoker.e2e.Clear_ShowVotes_ShareSessionLink;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ShowVotesButtonTest {
+    private final WebDriver driver = getDriver();
+    private MainPage mainPage = new MainPage(driver);
+    private SessionPage sessionPage = new SessionPage(driver);
+    final String username = "michal";
```

Comment:
Constant should be a `static` variable:

```suggestion
    private static final String USERNAME = "michal";
```